doc/man3/SSL_groupid_to_name.pod

```diff
@@ -0,0 +1,42 @@
+=pod
+
+=head1 NAME
+
+SSL_groupid_to_name - get name of group
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ const char *SSL_groupid_to_name(const SSL *ssl, int group_id);
+
+=head1 DESCRIPTION
+
+SSL_groupid_to_name() is used to retrieve the algorithm name
```

Comment:
This is my biggest concern about this PR. This returns the algorithm name which is actually not that interesting by itself. We want the real group name to be presented. I think either the tlsname or realname of the group would be more useful.

Reply:
It should always be the tlsname IMO.

Reply:
Done.